liveblog.php

```diff
@@ -252,7 +252,16 @@ public static function init() {
 			 * we can possibly introduce this to other post types later.
 			 */
 			$post_types = array( 'post' );
-			self::$supported_post_types = apply_filters( 'liveblog_modify_supported_post_types', $post_types );
+
+			/**
+			 * Filters post types supported by liveblog.
```

Comment:
There is a plan to possibly change the name of the plugin, so maybe change `liveblog` here to be `this plugin`?

Reply:
Are we changing the name in this release?
If not, I think this should stay as `liveblog` as it'll be easier to change the prefix with a search-replace when the name is finalised.

Reply:
what do you think?

Reply:
The prefix on the filter is unlikely to change, unless it was aiming to break BC. 

In this case, it was just the name in the filter description (line 257), that I was referring to.

Not a biggie - there are likely other instances of "liveblog" in other DocBlocks as well that would need changing.

Reply:
ah, got it! thanks!